Cargo.toml

```diff
@@ -12,6 +12,7 @@ exclude = ["cmd/crates/soroban-test/tests/fixtures/hello"]
 
 [workspace.package]
 version = "20.1.1"
+rust-version = "1.74.0"
```

Comment:
The packages in this repo claimed to minimally supported rust 1.69, 1.70, and 1.72, however they actually require 1.74. This is part of why we're adding these new CI runs that test against the msrv. The new CI runs failed because of this. The crates in this repo require 1.74 because they're dependent on other crates that have a msrv of 1.74.